### DISCUSSION TITLE
在src内写测试文件(*.test.ts)，提示 test expect can't find

### DISCUSSION THREAD

**Discussion by qieqie7** [umijs/umi#6429] [Q&A] [answered]
在src内写测试文件(*.test.ts)，提示 test expect can't find

请问这个问题应该怎么改，有什么相关的文章可以推荐看看嘛

**Comment by xiaohuoni** [comment, marked as answer]
应该会忽略 test 文件才对？

**Reply by qieqie7** [reply]
可能是我使用的pnpm安装的关系，无法使用隐式依赖，我装了一下 @types/jest 就好了。

**Reply by qieqie7** [reply]
你这个忽略怎么看出来的，我没有在test文件夹下面建立，是.test.ts文件

**Reply by xiaohuoni** [reply]
我说的就是 .test.ts 文件，执行dev的时候，会忽略这些文件。
